google/cloud/aiplatform/experimental/vertex_model/README.rst

```diff
@@ -90,6 +92,27 @@ To use the VertexModel class, your implementation must adhere to the following c
      def predict(self, data):
        return self.forward(data)
 
+     # Implementation of predict_payload_to_predict_input(), which converts a predict_payload object to predict() inputs
+     def predict_payload_to_predict_input(self, instances):
+       feature_columns = ['feat_1', 'feat_2']
+       data = pd.DataFrame(instances, columns=feature_columns)
```

Comment:
Based on the predict method it seems like this should be converted to torch tensor.